main/gpio_bitaxe.h

```diff
@@ -0,0 +1,11 @@
+#ifndef GPIO_BITAXE_H_
```

Comment:
My only comment is that it feels backwards that components would depend on a header in the app (main).